src/drivers/vmount/vmount_params.c

```diff
@@ -156,3 +157,71 @@ PARAM_DEFINE_INT32(MNT_MAN_PITCH, 0);
 */
 PARAM_DEFINE_INT32(MNT_MAN_YAW, 0);
 
+/**
+* Stabilize the mount (true for servo gimbal, false for passthrough).
+* Only affects RC input; Mavlink input can set this with the DO_MOUNT_CONFIGURE command.
```

Comment:
Actually, how about this parameter also sets the default on InputMavlinkCmdMount for consistency? (@ndepal)

Reply:
Good idea.